apps/src/weblab2/Weblab2View.tsx

```diff
@@ -1,18 +1,18 @@
-import React, {useState} from 'react';
-
-import './styles/Weblab2View.css';
-
-import {Config} from './Config';
+import './styles/Weblab2View.css'; // eslint-disable-line import/order
```

Comment:
can you also add a comment explaining why this is necessary, and which specifically of the other imports in the file we need this to before?